Src/FluentAssertions/Equivalency/Steps/GenericDictionaryEquivalencyStep.cs

```diff
@@ -35,14 +43,25 @@ public class GenericDictionaryEquivalencyStep : IEquivalencyStep
         return EquivalencyResult.ContinueWithNext;
     }
 
+    private static bool SubjectIsNonGenericDictionary(object subject)
```

Comment:
Here I followed the naming like `AssertSubjectIsNotNull` below, but I think this is a bit redundant.

So I would probably rename this to `IsNonGenericDictionary()`.

And I propose to change the method name below also to something like: `AssertIsNotNull()` or simply `IsNotNull()`.

(Or alternatively pass in the comparands object rather than `comparands.Subject`  🤔 )

What do you think?

Reply:
> So I would probably rename this to IsNonGenericDictionary().

Yeah, that is a better name